en/mkdocs.yml

```diff
@@ -89,8 +89,8 @@ nav:
         - API Schema Validation: how-tos/api-schema-validation.md
         - HTTP 2.0: how-tos/http-2.0.md
         - Rate Limiting:
-            - Distributed Throttling: how-tos/rate-limiting/distributed-throttling.md
-            - Adding Throttling Policies: how-tos/rate-limiting/adding-throttling-policies.md
+            - Enabling distributed throttling: how-tos/rate-limiting/distributed-throttling.md
+            - Enabling local throttling: how-tos/rate-limiting/adding-throttling-policies.md
```

Comment:
We need to add redirects to this links. otherwise old links will not work

Reply:
Links are same as the previous. I only changed the Title of the document.